ui/env-prod.sh

```diff
@@ -0,0 +1,5 @@
+
+pattern='  window.__env.apiUrl.*'
+replacement="  window.__env.apiUrl = '${apiUrl}';"
+
+sed -i -e 's,'"$pattern"','"$replacement"',' /usr/share/nginx/html/env.js
```

Comment:
is it not possible to move this to a `.json` file and patch that accordingly or change it accordingly? What is the reason for patching a `.js` file?

Reply:
@JaDogg Actually I used the javascript file which is imported in static HTML file so that whenever the homepage is loaded, the script is run and the variables are as well loaded.